flink-connectors/flink-connector-kinesis/src/main/java/org/apache/flink/streaming/connectors/kinesis/config/ConsumerConfigConstants.java

```diff
@@ -143,6 +128,13 @@ public SentinelSequenceNumber toSentinelSequenceNumber() {
 	/** The interval after which to consider a shard idle for purposes of watermark generation. */
 	public static final String SHARD_IDLE_INTERVAL_MILLIS = "flink.shard.idle.interval";
 
+	/**
+	 * Boolean to indicate whether to compare/enforce shardId format based on the one defined in
+	 * DynamodbStreamsShardHandle.
+	 */
+	public static final String DYNAMODB_STREAMS_SHARDID_FORMAT_CHECK =
```

Comment:
This property isn't actually used? If so, please remove.

Reply:
The key is never used to set the property, that's what not used refers to. The documentation also does not mention why anyone would use it. If there is no use case for this, let's remove it.

Reply:
changed the default to `true` and removed the config property.